grasa_event_locator/templates/allEvents.php

```diff
@@ -40,14 +41,14 @@
                       </div>
                       <div class="modal-footer">
                         <button type="button" class="btn btn-secondary" data-dismiss="modal">Never Mind</button>
-                        <button type="button" class="btn btn-danger" data-dismiss="modal" onclick="">Confirm Delete</button>
+                        <button type="button" class="btn btn-danger" data-dismiss="modal" onclick="window.location.href='{% url 'deny_event' p.id %}'">Confirm Delete</button>
```

Comment:
This button deletes the event, not just deny it, right? I was wondering about the method name.